lib/controllers/platform-controller.ts

```diff
@@ -143,6 +143,21 @@ export class PlatformController implements IPlatformController {
 		const hasPlatformDirectory = this.$fs.exists(
 			path.join(projectData.platformsDir, platformName)
 		);
+
+		if (hasPlatformDirectory) {
+			const isPlatformDirectoryEmpty =
```

Comment:
Since we are checking the number of items in the platform directory, the variable name is a bit misleading. I guess we should call it something like `platformDirectoryItemCount`, and move the `<= 5` into the `if` statement below?